.github/workflows/cml.yaml

```diff
@@ -4,20 +4,29 @@ jobs:
   run:
     runs-on: [ubuntu-latest]
     container: docker://dvcorg/cml:0-dvc2-base1
+    env:
+      GIT_USER: dvc
+      GIT_EMAIL: dvc@iterative.ai
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
+          ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0
+      - name: Fix git safe.directory in container
+        run: |
+          git config --global --add safe.directory ${GITHUB_WORKSPACE}
```

Comment:
Is this required because of `dvc get`?

Reply:
It's required for `dvc exp run`

Reply:
Is it needed for `dvc repro` too? Can't find any mention of this requirement in https://dvc.org/doc. Asking because we may need to update/fix https://github.com/iterative/cml_dvc_case/blob/master/.github/workflows/cml.yaml

Reply:
For `dvc repro` normally the answer is no, but it depends on whether or not the pipeline includes imports (since the import source repo may need to be cloned depending on the repo state)